### PR TITLE
wasmvm beta 3

### PR DESCRIPTION
**Pull request by faddat**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,7 +30,7 @@ require (
 	github.com/CosmWasm/wasmvm v1.0.0-soon2.0.20211209162446-b96c92d21aaf // indirect
```
Comment (giansalex) [inline go.mod:30]: should be `github.com/CosmWasm/wasmvm v1.0.0-beta3` ?

Comment (faddat) [reply go.mod:30]: nope, it is go doing its tag breaking thing.  but let me check the commit height to be certain.

Comment (faddat) [reply go.mod:30]: ah.  

More fixing needed.

Comment (faddat) [reply go.mod:30]: oop.  That would have broken everything...

**Comment by faddat** [comment]
night everyone.  Never actually seen it require an explicit update for an indirect dependency.

**Review by JakeHartnell — APPROVED** [review]

**Review by JakeHartnell — CHANGES_REQUESTED** [review]
Ok, this is sadly not building...

**Pull request closed without merging**